python/py-stagger/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-stagger
+
+github.setup        staggerpkg stagger 1.0.1 release-
+github.tarball_from archive
+
+revision            0
+maintainers         nomaintainer
+platforms           {darwin any}
+supported_archs     noarch
+license             BSD
+
+description         ID3v1/ID3v2 tag manipulation package in pure Python 3
+long_description    The ID3v2 tag format is notorious for its useless \
+                    specification documents and its quirky, mutually \
+                    incompatible part-implementations. Stagger is to provide a \
+                    robust tagging package that is able to handle all the \
+                    various badly formatted tags out there and allow you to \
+                    convert them to a consensus format.
+
+checksums           rmd160  937a32fc9cc8898e9c60d8101af563089949d173 \
+                    sha256  c7226bc779602850f497577e2f554136c888485b63789c77d3816dc6d223dda0 \
+                    size    5111466
+python.versions     39 310 311 312 313
```

Comment:
can we only add recent Python versions please for new ports (see [here](https://trac.macports.org/wiki/Python#PythoninMacPorts)); I would recommend `312 313`

Reply:
@reneeotten https://github.com/macports/macports-ports/pull/26385#discussion_r1824901881:
> can we only add recent Python versions please for new ports (see [here](https://trac.macports.org/wiki/Python#PythoninMacPorts)); I would recommend `312 313`

Sure, updated.